src/main/java/application/YariEditor.java

```diff
@@ -54,7 +54,6 @@ public void start(Stage primaryStage) throws Exception {
         Scene splashScene = new Scene(welcomeSplash);
         ThemeUtil.setThemeOnScene(splashScene);
 
-        primaryStage.initStyle(StageStyle.UNDECORATED);
```

Comment:
Was this redundant? The FX scenes were designed to be undecorated.

Reply:
It was not redundant and did what it was intended to do, however, our *real life* decision tables were too big to see in the static (kinda small width) size of the application.  Removing this line gets us back to a *standard* application that is resizable and can be maximized for viewing large decision tables.

Reply:
Btw, rosybrown is in the spectrum of Java colors and has a better contrast with fonts than the darkred in my opinion--tell me what you think.  I'm totally good with you changing it back. ;)  As a side note, I'm going to push updates to the Yari code/etc. as soon as I can...maybe start a new Github project or organization?

Reply:
Resizing of the main undecorated stage is implemented. You should be able to drag from the corners or sides as on decorated stages to resize the application.

I suppose the real issue here is the fact that this functionality is not obvious enough!

Reply:
I did try it before I removed that line but I couldn't get it to resize--I might not have been clicking my mouse button hard enough ¯\_(ツ)_/¯

Reply:
Haha! I'll look at it and see if perhaps something broke the resizing. It's definitely working in the 2.0.2 release though, just tested. I intend to do some more visual optimization for large tables, I just haven't gotten to it. A lot of the project was put together fairly quickly so I know there are improvements to be made throughout. If you run into any issues or feature ideas while using the applications, let me know or create an issue here and I'll get to it.

I'll take a look at the rosybrown. I'm up for changes. I was just interested in the reasoning :). Always learning!

I personally like GitHub better than BitBucket (TravisCI and other integrations), but whatever you go with is fine with me. If you go with GitHub, it would be ideal if we can group them together. Let me know if you need any permissions to this repository to accomplish this.

Reply:
Also, going back to your original reply; do you believe a maximize button should be added to the UI?

Reply:
If staying with the undecorated theme, yes.

Reply:
If staying with the undecorated theme, yes.